src/test/resources/features/Device.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Testing Device API
+
+  Scenario Outline: Create Device
+    Given I want to create a device
+    When I create a device
+    Then I should get a response with status <code>
+    And I should get a response with body
+
+    Examples:
+      | code |
+      | 200  |
+
+  Scenario Outline: Create Same Device
+    Given I want to create a device
+    When I create a device
+    Then I should get a response with status <code>
+
+    Examples:
+      | code |
+      | 409  |
+
+  Scenario Outline: Get Devices
+    Given I want to get all devices
+    When I get all devices
+    Then I should get a response with status <code>
```

Comment:
It could be nice to validate the content of the response here

Reply:
fixed

Reply:
I got confused because the code moved up but the comment didn't, oops. Nice work!